Sources/Confidence/Confidence.swift

```diff
@@ -128,9 +129,14 @@ public class Confidence: ConfidenceEventSender {
                     guard let self = self else {
                         return
                     }
-                    self.track(eventName: event.name, message: event.message)
-                    if event.shouldFlush {
-                        eventSenderEngine.flush()
+                    do {
+                        try self.track(eventName: event.name, message: event.message)
+                        if event.shouldFlush {
+                            eventSenderEngine.flush()
+                        }
+                    } catch {
+                        Logger(subsystem: "com.confidence", category: "track").debug(
+                            "Error from EventProducer, failed to track event: \(event.name)")
```

Comment:
Good choice of catching and logging here!
But I think this type of issue merits a _warning level_ wdyt?